mlir/lib/ExecutionEngine/ROCm/CMakeLists.txt

```diff
@@ -55,6 +55,13 @@ target_link_libraries(rocm-runtime-wrappers
   LLVMSupport
   ${ROCM_RUNTIME_LIBRARY}
 )
+# Suppress some compiler warnings when building with clang.  The first two
```

Comment:
Duplicates #346 but a different way.

Reply:
... yeah, that's actually a better way to do it